src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioDocumentTrackingService.cs

```diff
@@ -136,11 +133,20 @@ public int OnCmdUIContextChanged([ComAliasName("Microsoft.VisualStudio.Shell.Int
 
         public event EventHandler<EventArgs> NonRoslynBufferTextChanged;
 
-        public void OnNonRoslynViewOpened(ITextView view)
+        public void OnNonRoslynBufferOpened(ITextBuffer buffer)
+        {
+            buffer.Changed += OnNonRoslynBufferChanged;
```

Comment:
You may wish to consider PostChanged. If changes happen in response to other changes (consider typing inside inline rename), you'll get one event instead of many.